policy/modules/system/systemd.te

```diff
@@ -1306,6 +1326,8 @@ files_manage_var_lib_dirs(systemd_tmpfiles_t)
 files_purge_tmp(systemd_tmpfiles_t)
 files_read_etc_files(systemd_tmpfiles_t)
 files_read_etc_runtime_files(systemd_tmpfiles_t)
+files_relabel_config_dirs(systemd_tmpfiles_t)
+files_relabel_config_files(systemd_tmpfiles_t)
```

Comment:
It seems like this should also go in the factory tunable.

Reply:
> It seems like this should also go in the factory tunable.

Not necessarily.

According to my observations systemd will always try to restorecon the files, even if the target files exist and there is no need to copy them from the factory dir.

An alternative would be to put them - as you requested - into the factory conditional and put a dontaudit rule in the else section.

Reply:
If you're not using the factory, then why would tmpfiles be acting on config files? Most general purpose systems have persistent config files and probably wouldn't be having these tmpfiles entries.